release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -461,17 +461,17 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=20455
 After cluster installation, if you are using the OpenShift SDN cluster network provider or the OVN-Kubernetes cluster network provider, you can change your hardware MTU and your cluster network MTU values. Changing the MTU across the cluster is disruptive and requires that each node is rebooted several times. For more information, see xref:../networking/changing-cluster-network-mtu.adoc#changing-cluster-network-mtu[Changing the cluster network MTU].
 
 [id="ocp-4-10-redfish-events"]
-==== Low-latency Redfish hardware event delivery (Developer Preview)
+==== Bare-metal events (Technology Preview)
```

Comment:
@aireilly Could you please add an xref to the main docs?

Reply:
done